framework/behaviors/SluggableBehavior.php

```diff
@@ -81,13 +83,13 @@ class SluggableBehavior extends AttributeBehavior
      */
     public $value;
     /**
-     * @var bool whether to generate a new slug if it has already been generated before.
+     * @var boolean whether to generate a new slug if it has already been generated before.
```

Comment:
This is wrong change. Should be `bool` because `boolean` can't be used as return type in PHP 7+.